### DISCUSSION TITLE
Dataset folders

### DISCUSSION THREAD

**Discussion by OliviaOliveiira** [bghira/SimpleTuner#570] [Q&A]
Hey!
I was wondering if SimpleTuner supports text files and datasets from different folders 
e.g. Instance dir /Dataset
/Dataset/images123
/Dataset/images456
and in each of those folders would be text files with captions (might be the same name (0.png, 0.txt)

**Comment by bghira** [comment]
see DATALOADER.md file

**Reply by OliviaOliveiira** [reply]
Decided to try it rightaway to test it myself regardless of the outcome, and it did work the way I expected it to, thanks!)
